setup.py

```diff
@@ -19,6 +19,14 @@
         "mdformat-gfm>=0.3.5",  # Needed for formatting GitHub-flavored markdown
         "mdformat-frontmatter>=0.4.1",  # Needed for frontmatters-style headers in issue templates
```

Comment:
i think you need to add `"mdformat-pyproject>=0.0.1"` here

Reply:
Addressed here: https://github.com/ApeWorX/silverback/pull/36

Reply:
btw it seems like it merged without this